From be7ba6a84c49d6677ebc6c94a3d97849a4ddb183 Mon Sep 17 00:00:00 2001
From: Marcin Lewandowski <marcin.lewandowski@swmansion.com>
Date: Thu, 26 Mar 2020 14:29:26 +0100
Subject: [PATCH 1/1] Use direct, not fanout exchange type in RabbitMQ

---
 transports/janus_rabbitmq.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/transports/janus_rabbitmq.c b/transports/janus_rabbitmq.c
index 5437d61b..43ec1875 100644
--- a/transports/janus_rabbitmq.c
+++ b/transports/janus_rabbitmq.c
@@ -110,7 +110,7 @@ static gboolean rmq_admin_api_enabled = FALSE;
 static gboolean notify_events = TRUE;
 
 /* FIXME: Should it be configurable? */
-#define JANUS_RABBITMQ_EXCHANGE_TYPE "fanout"
+#define JANUS_RABBITMQ_EXCHANGE_TYPE "direct"
 
 /* JSON serialization options */
 static size_t json_format = JSON_INDENT(3) | JSON_PRESERVE_ORDER;
-- 
2.24.1 (Apple Git-126)

